### DISCUSSION TITLE
Отключить вывод цены доставки в виджете

### DISCUSSION THREAD

**Discussion by AndreyVasilkov** [cdek-it/widget#2] [Q&A] [answered]
Добрый день. 
Используем 3 версию виджета, как можно отключить вывод стоимости доставки у ПВЗ и Постаматов?

**Comment by vermorag** [comment, marked as answer]
Чтобы виджет не показывал стоимость достаточно использовать версию [^3.2](https://github.com/cdek-it/widget/releases/tag/v3.2.0) и передать при инициализации параметр `canChoose=true` и не передать `from`. Тогда виджет будет отображать только кнопку выбора без информации о стоимости. Но эта же логика применится и к доставке до двери.
